backend/app/views/spree/admin/product_properties/index.html.erb

```diff
@@ -7,7 +7,9 @@
   <% if can?(:create, Spree::ProductProperty) %>
     <ul class="tollbar inline-menu">
       <li>
-        <%= link_to_add_fields t('spree.add_product_properties'), 'tbody#product_properties', class: 'plus btn btn-primary' %>
+        <%= button_tag t('spree.add_product_properties'), type: 'button',
+                    data: { target: 'tbody#product_properties' },
```

Comment:
Why this indent? I think two spaces from the start of the `button_tag` is the recommended way.

Reply:
ok, we're getting there. I had the idea to put the arguments in one column (before the change to button_tag)